### PR TITLE
Add remove Feature

### PR DESCRIPTION
**Pull request by mwanawabangona**
- Add Html & Css
- Add JavaScript
- Add Weppack
- Add Linters to fix errors

### CODE REVIEW

File: src/todolist.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          completed: false,
+          index: 3,
+        },
+        {
+          description: 'test4',
+          completed: false,
+          index: 4,
+        },
+      ];
+    }
```
Comment (BrianSammit) [inline src/todolist.js]: - [x] Kindly delete all hardcoded items from the tasks array as this is one of the requirements for this milestone. 
😄 💪

**Review by BrianSammit — CHANGES_REQUESTED** [review]
# CHANGES REQUIRED 🔴 🛑
Hello @mwanawabangona, great job you are doing so far in this project. 💯
TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 
- Your CRUD actions are working perfectly 

However, there are a few changes to make to move to the next project 🛠️ . Keep working hard 💪 💻 

## Required Changes ♻️

- [x]  It would be better if you add e`slintrc.json` and `stylelontrc.json` to .gitignone because these files are to test the linter locally but aren't needed to run the project. Kindly add these files to .gitignore.
See my inline comment to know what changes are needed. 👇 👀 

Please make these required changes and send it again for another review 👍 💪
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# HAPPY CODING!!! 😄 

cat


**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

--------

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/todolist.js (hunk 1 above)
Comment (mwanawabangona) [reply src/todolist.js]: done

File: src/todolist.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+export default class TodoList {a
```
Comment (Dipeshtwis) [inline src/todolist.js:1]: - Optional:-This ' a ' may be a typo error. Please remove this.

**Review by Dipeshtwis — APPROVED** [review]
# Hi @mwanawabangona , 

### Status: Approved ✔️

:+1:  Professional repo, with good commit message and PR title, keep up the good work :muscle:
:+1: The project setup is well organized. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- See below :small_blue_diamond: 


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**